packages/airnode-node/src/workers/local-gateways/server.ts

```diff
@@ -18,6 +17,7 @@ const httpSignedDataBodySchema = z.object({
   encodedParameters: z.string(),
 });
 const DEFAULT_PORT = 3000;
+const DEFAULT_PATH_KEY = '05701bc4-4eb4-4f60-b4eb-075c80ea98c6';
```

Comment:
Oh, you've chosen a true random UUID. I was thinking more like a "fake" UUID so nobody will wonder whether it should be configured somewhere. Something like `01234567-abcd-abcd-abcd-012345678abc` or `ffffffff-ffff-ffff-ffff-ffffffffffff`, you know what I mean.

Reply:
Ah yeah I just switched the old `API_KEY` into the path :sweat_smile:. But you make a great point, I'll update :smile: